README.md

```diff
@@ -429,28 +436,27 @@ spec:
           processors: []
           exporters: [logging]
 ```
+The usage of `$$` in the replacement keys in the example above is based on the information provided in the Prometheus receiver [README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md) documentation, which states:
+`Note: Since the collector configuration supports env variable substitution $ characters in your prometheus configuration are interpreted as environment variables. If you want to use $ characters in your prometheus configuration, you must escape them using $$.`
 
 Behind the scenes, the OpenTelemetry Operator will convert the Collector’s configuration after the reconciliation into the following:
 
 ```yaml
     receivers:
       prometheus:
         config:
-          global:
-            scrape_interval: 1m
-            scrape_timeout: 10s
-            evaluation_interval: 1m
           scrape_configs:
           - job_name: otel-collector
-            honor_timestamps: true
             scrape_interval: 10s
-            scrape_timeout: 10s
-            metrics_path: /metrics
-            scheme: http
-            follow_redirects: true
```

Comment:
Shouldn't these values be preserved in the configuration given to the collector?  These are scrape-time options that are needed by the receiver.